python/google/protobuf/internal/python_protobuf.cc

```diff
@@ -30,7 +30,7 @@
 
 // Author: qrczak@google.com (Marcin Kowalczyk)
 
-#include <google/protobuf/python/python_protobuf.h>
```

Comment:
@georgthegreat I'm trying to understand how this even worked before. It seems like it should have been a compile error. @anandolee Do you know if this file is still used? I think this file might not even be getting built.

Reply:
I do not know.
We have had this patch for many years now.

Reply:
From I very brief check I can say that this file is not needed any more.
If you could confirm it, I can change this PR to remove the file (and corresponding header) instead. Or you can do this by yourself.

Reply:
Sounds good to me to just delete the files, but @anandolee can probably say for sure whether we still need them.